### DISCUSSION TITLE
Actual Budget post-install

### DISCUSSION THREAD

**Discussion by LapplandsCohan** [community-scripts/ProxmoxVE#74] [Q&A]
This thread is a continuation of [the original Actual Budget thread](https://github.com/tteck/Proxmox/discussions/2727).

There are several posts stating that to get Actual Budget to work you have to set up a reverse proxy.

- Do you set up a separate LXC for the reverse proxy, or do you install reverse proxy software on the Actual Budget LXC?

**Comment by Mellowlynx** [comment]
Best way is a separate LXC.
You can use it for all your apps, so if you also deploy a Nginx Proxy Manager LXC, you can set it up.
